lib/index.js

```diff
@@ -26,6 +26,7 @@ Plotly.register([
     require('./scatter3d'),
     require('./surface'),
     require('./isosurface'),
+    require('./volume4d'),
```

Comment:
`volume4d` interesting. Why do you prefer this over `volume`?

Reply:
I think `volume3d` & to some extends `volume` both refer to three-dimensions which this trace is not really about (i.e. exploring the data in 4 dimensions). If the value was declared as a `vector` instead of `scalar` (case of plotting wind directions & speed) the data could be 6d/7d.